ci/get-engine.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+if [[ -n "${DEBUG-}" ]]; then
+    set -x
+fi
+
+pushd "$(dirname "$0")"
```

Comment:
I think for all files which are using ordered input args we should have a comment stating what each one is, for maintainability.